internal/test/e2e/databasesql/main.go

```diff
@@ -72,20 +71,19 @@ func NewServer() *Server {
 	}
 }
 
-func (s *Server) queryDb(w http.ResponseWriter, req *http.Request) {
+func (s *Server) query(w http.ResponseWriter, req *http.Request, query string) {
 	ctx := req.Context()
-
 	conn, err := s.db.Conn(ctx)
 	if err != nil {
 		panic(err)
 	}
 
-	rows, err := conn.QueryContext(req.Context(), sqlQuery)
+	rows, err := conn.QueryContext(req.Context(), query)
 	if err != nil {
-		panic(err)
+		logger.Error(err.Error())
 	}
 
-	logger.Info("queryDb called")
+	logger.Info(fmt.Sprintf("Query: %s", query))
```

Comment:
nit: use the structured elements of the logger to handle this.

```suggestion
	logger.Info("queryDB called", "query", query))
```